codecov.yml

```
@@ -1,6 +1,7 @@
 coverage:
   ignore:
     - docs/*
+    - packges/ability-attributes/src/*
```

```suggestion
    - packages/ability-attributes/src/*
```

Let's fix this typo, I will fix others in a separate PR. Good catch 👍